pipelines/zap/pipeline.yml

```diff
@@ -88,9 +88,9 @@ jobs:
       put: slack
       params:
         channel: <%= project['slack_channel'] ? "'##{project['slack_channel']}'" : "{{slack-channel}}" %>
-        icon_url: {{slack-icon}}
```

Comment:
I would just take out the configuration altogether. Convention over configuration 😉